archinstall/lib/disk.py

```diff
@@ -25,7 +25,7 @@ def __init__(self, path, info=None):
 		self.path = path
 		self.info = info
 		self.part_cache = OrderedDict()
-		# TODO: Currently disk encryption is a BIT missleading.
+		# TODO: Currently disk encryption is a BIT misleading.
 		#       It's actually partition-encryption, but for future-proofing this
 		#       I'm placing the encryption password on a BlockDevice level.
 		self.encryption_passwoed = None
```

Comment:
`encryption_password`?

Reply:
This makes me a bit happy, that even the best of tools don't catch some spelling mistakes ^^ Nicely spotted @pyfisch